deploy/examples/operator-openshift.yaml

```
@@ -749,6 +749,16 @@ spec:
                 configMapKeyRef:
                   key: ROOK_CSI_ENABLE_NFS
                   name: ocs-operator-config
+            - name: ROOK_CSI_ENABLE_CEPHFS
+              valueFrom:
+                configMapKeyRef:
+                  key: ROOK_CSI_ENABLE_CEPHFS
+                  name: ocs-operator-config
+            - name: ROOK_CSI_ENABLE_RBD
+              valueFrom:
+                configMapKeyRef:
+                  key: ROOK_CSI_ENABLE_RBD
+                  name: ocs-operator-config
```

```suggestion
            - name: ROOK_CSI_ENABLE_CEPHFS
              valueFrom:
                configMapKeyRef:
                  key: ROOK_CSI_ENABLE_CEPHFS
                  name: ocs-operator-config
            - name: ROOK_CSI_ENABLE_RBD
              valueFrom:
                configMapKeyRef:
                  key: ROOK_CSI_ENABLE_RBD
                  name: ocs-operator-config
```

do we read these values from ocs-operator-config? We have the boolean setting in rook-ceph-operator-configmap for these similar to enabling csi-addons

---

from next release ocs-operator will set these values to "false" if odf is running in provider mode and ocs-client-operator manages the CSI.

---

from next release mean 4.16 or 4.17 :)?

---

4.16.